src/momento/responses/scalar_data/delete.py

```diff
@@ -20,19 +17,12 @@ class CacheDeleteResponse(CacheResponse):
 class CacheDelete(ABC):
     """Groups all `CacheDeleteResponse` derived types under a common namespace."""
 
-    @dataclass
```

Comment:
dataclass also provided equals and hash methods. Do we have a substitute for that here?

Reply:
Sounds nice, but that hasn't been a consideration for responses that I'm aware of and it isn't tested. I can put the @dataclass declarations back, I don't know that it's done consistently.

Reply:
Added in [0ad0e4b](https://github.com/momentohq/client-sdk-python/pull/258/commits/0ad0e4b1a88e27eab579900db45f97e253211e4f) and [aae5545](https://github.com/momentohq/client-sdk-python/pull/258/commits/aae5545c6e699ea8bc0ef69f2d5b42d29e41bab3)